### PR TITLE
Finished Refactor

### PR DESCRIPTION
**Pull request by AlecThomasQuinn**

### CODE REVIEW

File: assignments/lambda-classes.js [hunk 1]
```diff
@@ -1 +1,124 @@
...
+
+//Person, Instructor, Student, PM
+
+// Person
+// name, age, location
+// speak()
+
+class Person{
+    constructor(propsPerson){
+        this.name = propsPerson.name
```
Comment (AnnaTownsend9916) [inline assignments/lambda-classes.js:11]: make sure you are using semantical code and including your semi colons after each list

File: assignments/lambda-classes.js [hunk 2]
```diff
@@ -1 +1,124 @@
...
+class Student extends Person{
+    constructor(propsPerson){
+        super(propsStudent)
+        this.previousBackground = propsStudent.previousBackground
+        this.className = propsStudent.className
+        this.favSubjects = propsStudent.favSubjects
+    }
+
+    //listsSubjects(){}
+
```
Comment (AnnaTownsend9916) [inline assignments/lambda-classes.js:58]: There should be a for method here

File: assignments/lambda-classes.js [hunk 3]
```diff
@@ -1 +1,124 @@
...
+})
+
+const Anna = new PM({
+    name: 'Anna Townsend',
+    age: 'it is impolite to ask',
+    location: 'Im not googling that thats weird',
+
+    specialty: 'closing zoom sessions too early',
+    favLanguage: 'Front-end stuff',
+    catchPhrase: 'Bee Ayy In Ayy In Ayy Ess',
```
Comment (AnnaTownsend9916) [inline assignments/lambda-classes.js:119]: I had to say this out loud haha

**Review by AnnaTownsend9916 — COMMENTED** [review]
Great work! Make sure you are including your semi colons properly. Otherwise in the future some of your code will collect errors and not run properly because you are missing one or more. You're doing great so far, make sure to include that for method as well

File: assignments/lambda-classes.js (hunk 3 above)
Comment (AlecThomasQuinn) [reply assignments/lambda-classes.js:119]: All according to plan.

**Pull request merged**